CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ set(OPENSMT_VERSION_MAJOR 2)
 set(OPENSMT_VERSION_MINOR 8)
 set(OPENSMT_VERSION_PATCH 1)
 set(OPENSMT_VERSION ${OPENSMT_VERSION_MAJOR}.${OPENSMT_VERSION_MINOR}.${OPENSMT_VERSION_PATCH})
+execute_process(
```

Comment:
Maybe we should move this script into `bin/util/CMakeLists.txt`?
`OPENSMT_GIT_DESCRIPTION` is only used there anyway...